java-squid/src/main/java/org/sonar/java/model/JavaTree.java

```diff
@@ -325,6 +384,8 @@ public void accept(TreeVisitor visitor) {
     private final Kind kind;
     @Nullable
     private final TypeTree bound;
+    // FIXME annotations can only by accessed through the children iterator
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1134)